src/renderer/components/dialogs/DeadDrop.js

```diff
@@ -26,16 +26,18 @@ class DeadDropDialog extends React.Component {
   }
 
   never () {
-    ipcRenderer.send('dispatch', 'blockContact', this.props.deadDropChat.contact.id)
+    var contactId = this.props.deadDrop.contact.id
+    ipcRenderer.send('dispatch', 'blockContact', contactId)
     this.close()
   }
 
   render () {
-    const { deadDropChat } = this.props
-    var nameAndAddr = deadDropChat && deadDropChat.deaddrop.contact && deadDropChat.deaddrop.contact.nameAndAddr
+    const { deadDrop } = this.props
+    var nameAndAddr = deadDrop && deadDrop.contact && deadDrop.contact.nameAndAddr
+
     const title = 'Chat request'
     const body = `Chat with ${nameAndAddr}?`
```

Comment:
These need translation I guess? Make a TODO for it for now if it's too much trouble.

Reply:
done